package.json

```diff
@@ -31,14 +31,14 @@
     "update": "ncu -u"
   },
   "peerDependencies": {
-    "@oruga-ui/oruga-next": "0.9.0"
+    "@oruga-ui/oruga-next": "~0.9.0"
   },
   "dependencies": {
     "bulma": "1.0.2"
   },
   "devDependencies": {
-    "@oruga-ui/examples": "0.9.0",
-    "@oruga-ui/oruga-next": "0.9.0",
+    "@oruga-ui/examples": "~0.9.0",
```

Comment:
here you can fix the version to a specfic one, because its only for dev purpose 
```
"@oruga-ui/examples": "0.9.1",
"@oruga-ui/oruga-next": "0.9.1",
```